sql/moz-fx-data-shared-prod/braze_derived/waitlists_v1/query.sql

```diff
@@ -0,0 +1,26 @@
+SELECT
+  email_id AS external_id,
```

Comment:
Should we consistently refer to `email_id` as either `email_id` or `external_id` in all tables?

Reply:
ah, yes...good point! since braze expects external_id, I'll use that :) ty